res/openvpn-gui-res-no.rc

```diff
@@ -273,7 +273,7 @@ BEGIN
     IDS_NFO_RECONN_FAILED "Tilkobling til %s feilet."
     IDS_NFO_STATE_SUSPENDED "Status: Hvilemodus"
     IDS_ERR_READ_STDOUT_PIPE "Feil under lesning fra OpenVPN StdOut pipe."
-    IDS_ERR_CREATE_EDIT_LOGWINDOW "Opprettelse av RichEdit LogWindow feilet!!"
+    IDS_ERR_CREATE_EDIT_LOGWINDOW "Opprettelse av RichEdit LogWindow feilet!"
```

Comment:
Is it possible to have a better description than "RichEdit LogWindow"?  I am no Windows developer so I have no deep understanding of what this really means.  Maybe @selvanair can provide some background?